java-components/cache/src/test/java/com/redhat/hacbs/artifactcache/ContainerRegistryTestResourceManager.java

```diff
@@ -153,7 +153,10 @@ private List<Path> createTestLayers() throws IOException {
             Files.writeString(shaFile, sha1);
         }
 
-        return List.of(layer1Path, layer2Path, layer3Path);
+        // HACK : OCIRepositoryClient assumes that the artifacts are in a directory and it then places them
+        //        within 'artifacts/...'. This is due to build-trusted-artifact changes as its storage stores
+        //        the contents of a directory not including the directory itself.
+        return List.of(layer1Path, layer2Path, layer3Path.resolve("com"));
```

Comment:
Not overly fond of this, but the current test doesn't match how oras stores (it stores the _contents_ of a directory versus the directory itself).